src/events/cd-event-list.vue

```diff
@@ -65,6 +72,18 @@
       getSessionListForEvent(event) {
         return event.sessions.map(session => session.name).join(', ');
       },
+      isEventFull(eventId) {
+        const currentEvent = this.events.find(event => event.id === eventId);
+        let totalEventCapacity = 0;
+        let totalTicketsBooked = 0;
+        forEach(currentEvent.sessions, (session) => {
```

Comment:
Probably better off just using the native `forEach` function of arrays instead of lodash.

```javascript
currentEvent.sessions.forEach((session) => {
  session.tickets.forEach((session) => {
    totalEventCapacity += ticket.quantity;
    totalTicketsBooked += ticket.approvedApplications;
  });
});
```

The less unnecessary dependencies, the better! 😅